hisat2/2.2.1/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM pgc-images.sbgenomics.com/d3b-bixu/samtools:1.15.1 AS build-env
```

Comment:
...I saw it on the internet? I did also question to myself whether I needed to do this. Since this syntax was used in the STAR Fusion dockerfile, I stuck with it

Reply:
So I think declaring an alias with no intent of using it is probably confusing. From the dockerfile ref:

`
Optionally a name can be given to a new build stage by adding AS name to the FROM instruction. The name can be used in subsequent FROM and COPY --from=<name> instructions to refer to the image built in this stage.
`

You are doing neither here. TGEN has a lot of good examples of how to leverage the alias; here's one: https://github.com/tgen/jetstream_containers/blob/main/tgen_phoenix/lancet/Dockerfile